eZ/Publish/Core/Persistence/Legacy/Content/FieldHandler.php

```diff
@@ -151,6 +151,10 @@ protected function getEmptyField(FieldDefinition $fieldDefinition, $languageCode
     public function createExistingFieldsInNewVersion(Content $content)
```

Comment:
Nice to have, since you've touched the method: 
```suggestion
    public function createExistingFieldsInNewVersion(Content $content): void
```